phpunit.xml.dist

```diff
@@ -22,6 +22,8 @@
         <!-- <server name="DOCTRINE_MONGODB_ODM" value="/path/to/doctrine-mongodb-odm/lib" /> -->
         <!-- <server name="DOCTRINE_MONGODB" value="/path/to/doctrine-mongodb/lib" /> -->
         <!-- <server name="DOCTRINE_COMMON" value="/path/to/doctrine-common/lib" /> -->
+        <!-- Allow 1 direct deprecation until https://github.com/doctrine/DoctrineMongoDBBundle/pull/675 is merged -->
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[direct]=1"/>
```

Comment:
In the meantime I've ignored the direct deprecation:

```
Remaining direct deprecation notices (1)

  1x: The "" service relies on the deprecated "Doctrine\Common\Cache\ArrayCache" class. It should either be deprecated or its implementation upgraded.
    1x in ServiceRepositoryTest::testRepositoryServiceWiring from Doctrine\Bundle\MongoDBBundle\Tests
```
so [this will be fixed](https://github.com/doctrine/DoctrineMongoDBBundle/pull/676#issuecomment-851326879) in https://github.com/doctrine/DoctrineMongoDBBundle/pull/675